media-libs/harfbuzz/metadata.xml

```diff
@@ -17,7 +17,11 @@
 		<remote-id type="cpe">cpe:/a:harfbuzz_project:harfbuzz</remote-id>
 	</upstream>
 	<use>
-		<flag name="glib">Compile with GLib Unicode support</flag>
-		<flag name="graphite">Use graphite to render complex non-Roman writing systems</flag>
+		<flag name="cairo">Use <pkg>x11-libs/cairo</pkg> graphics library</flag>
+		<flag name="experimental">Enable experimental APIs</flag>
+		<flag name="glib">Enable <pkg>dev-libs/glib</pkg> unicode functions</flag>
+		<flag name="graphite">Enable <pkg>media-gfx/graphite2</pkg> complementary shaper</flag>
+		<flag name="icu">Enable <pkg>dev-libs/icu</pkg> library unicode functions</flag>
+		<flag name="introspection">Enable <pkg>dev-libs/gobject-introspection</pkg> bindings</flag>
```

Comment:
There's no need to add `USE` descriptions for flags that already have a global description. Please only add `USE` flag descriptions for flags where `repoman` or `pkgdev` complains about missing `USE` description.